kihuha/iReporter
# #162904265 - Enable login functionality

#### What does this PR do?
Enables a user to login and receive an authentication token

#### Description of the task to be completed
Send a login request using the fetch API to the login endpoint at https://ireporter-kihuha.herokuapp.com/api/v2/users/login and set a cookie containing the received token on successful login. An appropriate error message should be displayed should the request fail.

#### How should this be manually tested?
Navigate to the [login page](https://kihuha.github.io/iReporter/ui/login.html), enter your username and password and click the 'confirm' button to send the request. On success, you will be redirected to the dashboard. On error, you will receive an error message.  

#### Relevant PT stories
https://www.pivotaltracker.com/story/show/162904265
